tuftool/tests/root_command.rs

```diff
@@ -130,6 +131,11 @@ fn check_signature_exists(root_json: &str, key_id: Decoded<Hex>) -> bool {
     return true;
 }
 
+fn check_version(root_json: &str) -> NonZeroU64 {
```

Comment:
should it be `get_root_version` ? function does not check for version and it cannot get version for any other role.
```suggestion
fn get_root_version(root_json: &str) -> NonZeroU64 {
```

Reply:
Yes, right. Will change it